docker/apm-server-proxy/server.js

```diff
@@ -0,0 +1,40 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+const util = require('util');
+const writeFile = util.promisify(fs.writeFile);
+
+const APM_SERVER_PORT =
+  parseInt(process.env.ELASTIC_APM_SERVER_PORT, 10) || 8200;
+const APM_SERVER_HOST = process.env.ELASTIC_APM_SERVER_HOST || '0.0.0.0';
+
+// App
+const app = express();
+app.use(bodyParser.raw({ type: 'application/x-ndjson' }));
+
+// used for health check
+app.get('/', (req, res) => {
+  console.log('Endpoint: /');
+  res.status(202).end();
+});
+
+const payloads = [];
+
+app.post('/intake/v2/events', (req, res) => {
```

Comment:
Instead of listening on a single endpoint we can also listen for all endpoints:

```
app.post('*', ...)
```

and write that to file containing the endpoint.